util-kotlin/src/main/kotlin/team/duckie/app/android/util/kotlin/exception.kt

```diff
@@ -0,0 +1,14 @@
+/*
+ * Designed and developed by Duckie Team, 2022
+ *
+ * Licensed under the MIT.
+ * Please see full license: https://github.com/duckie-team/duckie-android/blob/develop/LICENSE
+ */
+
+package team.duckie.app.android.util.kotlin
+
+class DuckieApiException(
+    override val message: String,
+    val code: String? = null,
+    val errors: List<String>? = null,
+) : Exception(message)
```

Comment:
override 되는 필드이므로 명시적 지정이 필요 없습니다.

```suggestion
) : Exception()
```